redis-utils-lib/src/cmr/redis_utils/config.clj

```diff
@@ -35,13 +63,40 @@
 
 (defconfig redis-retry-interval
   "The number of milliseconds to wait between Redis retry command."
-  {:default 100 :type Long})
+  {:default 400 :type Long})
 
 (defn redis-conn-opts
```

Comment:
All the new option lists are ops, so either this one needs to change or the others should end with 'opts'

Reply:
fixed.